plugins/home/src/homePageComponents/FeaturedDocs/FeaturedDocs.tsx

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import useAsync from 'react-use/lib/useAsync';
+import {
+  LinkButton,
+  EmptyState,
+  Link,
+  InfoCard,
+  Progress,
+  ErrorPanel,
+} from '@backstage/core-components';
+import { catalogApiRef, CatalogApi } from '@backstage/plugin-catalog-react';
+import { useApi } from '@backstage/core-plugin-api';
+import { EntityFilterQuery } from '@backstage/catalog-client';
+import { ClassNameMap } from '@material-ui/styles';
+
+import { makeStyles, Theme, Typography } from '@material-ui/core';
+
+/**
+ * Props customizing the <FeaturedDocs/> component.
+ *
+ * @public
+ */
+export type FeaturedDocsProps = {
+  /** The entity filter used to display only the intended item/s */
+  filter: EntityFilterQuery;
+  /** An optional color which can be customized through themes */
+  color?: 'inherit' | 'primary' | 'secondary' | undefined;
+  /** An optional ClassNameMap created with makeStyles */
+  customStyles?: ClassNameMap<string> | undefined;
+  /** An optional ReactNode for empty states */
+  emptyState?: React.ReactNode | undefined;
+  /** An optional path to set for entity entry  */
+  path?: string | undefined;
+  /** An optional limit to set for link destination  */
+  responseLimit?: number | undefined;
+  /** An optional string to customize sublink text */
+  subLinkText?: string | undefined;
+  /** An optional string or ReactNode to customize the card title */
+  title?: React.ReactNode | string | undefined;
+};
+
+const useStyles = makeStyles<Theme>(theme => ({
+  docDescription: {
+    fontSize: 16,
+    fontWeight: 400,
+    marginBottom: theme.spacing(2),
+    marginTop: theme.spacing(2),
+  },
+  docSubLink: {
+    fontSize: 12,
+    fontWeight: 700,
+    lineHeight: 2,
+  },
+  docsTitleLink: {
+    fontSize: 16,
+    fontWeight: 600,
+    lineHeight: 2,
+  },
+}));
+
+/**
+ * A component to display specific Featured Docs.
+ *
+ * @public
+ */
+export const FeaturedDocs = (props: FeaturedDocsProps) => {
+  const {
+    color,
+    customStyles,
+    emptyState,
+    filter,
+    path,
+    responseLimit,
+    subLinkText,
+    title,
+  } = props;
+  const linkText = subLinkText || 'LEARN MORE';
+  const defaultStyles = useStyles();
+  const styles = customStyles || defaultStyles;
```

Comment:
Like the intent of customizability. We try to support customisability the following way: https://backstage.io/docs/getting-started/app-custom-theme#overriding-backstage-and-material-ui-components-styles. For this to work follow the documentation adding a `Backstage...` class name.

Reply:
I added more variables in the styles above, but we feel that it increases the flexibility here to allow for `customStyles` to be passed in completely. Can we keep this here?

Reply:
Great. OK, let's keep the optional `customStyles`, but please make sure to add name for the component such that people can decide, which option they want to use to override the styles & don't have manually do it in React.

Reply:
@tudi2d I added `{ name: 'BackstageFeaturedDocs' },` 
Please let me know if you meant something else by them 👍